src/openrct2/config/Config.cpp

```diff
@@ -712,6 +713,21 @@ namespace Config
         }
         return std::string();
     }
+
+    static bool ExtractGogInstaller(const utf8* installerPath, const utf8* targetPath)
+    {
+        std::string path;
+        std::string output;
+
+        if (!Platform::FindApp("innoextract", &path))
```

Comment:
This function is only defined on Posix/Linux plataform headers, that's why all of the windows builds are failing.